### PR TITLE
Onramping transak

### PR DESCRIPTION
**Pull request by jo-chemla**
Adding transak-sdk to create an onramping widget on the v4-tsunami deposit UI. 
pooltogether-transak-onramp-en

### CODE REVIEW

**Comment by jo-chemla** [comment]
Added back the Help button in the middle of the three buttons below the deposit window, and moved the bridge tokens to the sidepanel in the settings! 

image

File: lib/components/Layout/PageHeader.tsx [hunk 1]
```diff
@@ -42,16 +47,50 @@ export const PageHeader = (props) => {
...
     <SettingsContainer t={t} className='ml-1 my-auto' title='Settings' sizeClassName='w-6 h-6'>
       <LanguagePicker />
       <ThemeSettingsItem t={t} />
+      <BridgeTokensModalTrigger prizePool={prizePool} />
       <TestnetSettingsItem t={t} />
     </SettingsContainer>
   )
 }
 
+const BUTTON_MIN_WIDTH = 100
```
Comment (dylandesrosier) [inline lib/components/Layout/PageHeader.tsx:62]: This is quite hidden. A better solution would be merging some of the cases (bridge & swap OR bridge & swap & buy) into a single option/modal. From there we could guide users properly. We could drop the iFrame for swapping as well and just link out for the swaps. Something like the following would be a step in the right direction:

- On deposit view there's "Get Tokens" & "Help"
- "Get Tokens" opens a modal with 3 clear sections: Bridge, Swap, Buy
- Bridge & Swap lead to links. Keeping networks in mind so the user ends up on a bridge that supports avalanche for example.
- Buy ultimately leads to the transak view, but has clear explanations on what tokens the user needs to buy.

File: lib/views/Deposit/DepositCard.tsx [hunk 2]
```diff
@@ -310,22 +310,55 @@ const HelpLink = () => {
   )
 }
 
-interface ExternalLinkProps {
-  prizePool: PrizePool
+const purple_default = '4c249f'
```
Comment (dylandesrosier) [inline lib/views/Deposit/DepositCard.tsx:313]: Could you pull this out into a separate location. A new file in `src/utils/services` makes sense to me.

**Review by dylandesrosier — CHANGES_REQUESTED** [review]
We have a very wide range of users, from crypto beginner to expert. We need to add some education into the flow rather than just giving them the buy crypto screen. There's a couple core users that should be accounted for:

Beginner - Needs to be taught about buying on the proper network, needs to buy both USDC and a gas token for that network.

Expert - Should be able to easily access the transak view. Explainers & guides shouldn't bury the transak view from them more than a click or two.

We wanted to spend some time designing a proper flow for this since it is a major update and can result in a lot of pain (imagine the case of a beginner coming and spending $1000 on USDC for Mainnet when they wanted to deposit on Avalanche). Unfortunately that's been deprioritized.

**Comment by dylandesrosier** [comment]
Could you also provide some more documentation (separate from this PR) on how you got the production API key and how that is going to be managed? I'm a little out of the loop on that process.

**Pull request closed without merging**